packages/block-editor/src/components/iframe/index.js

```diff
@@ -310,25 +311,34 @@ function Iframe(
 					);
 				}
 			) }
-			{ head }
 		</>
 	);
 
+	// Correct doctype is required to enable rendering in standards
+	// mode. Also preload the styles to avoid a flash of unstyled
+	// content.
+	const srcDoc = useMemo( () => {
+		return '<!doctype html>' + renderToString( styleAssets );
```

Comment:
Isn't it weird to have the styles inside the srcDoc prop?

Reply:
I don't even what that does?

Reply:
`srcDoc` sets the initial HTML. An iframe always has some initial HTML, even if we don't set anything, it will just be an empty head and body that we later replace. This sets the initial HTML to be the styles, after which we wait until the document is ready to render the content. This is to simulate normal page loading on the front end, where styles in the head block the rest of the content from rendering until they're all loaded.

Reply:
TIL